src/styles/constants.scss

```diff
@@ -2,6 +2,13 @@
  * Constants for stylesheet
  */
 
-$DESKTOP_WIDTH: 768px;
+$DESKTOP_WIDTH: 1240px;
+/**
+ * NOTE: this is completely arbitrary and loosely based upon values at
```

Comment:
Let's test on iPad (and maybe surface since i happen to have one) and see if this works